cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -658,12 +678,16 @@ public static Integer reviewId(HttpServletRequest request, boolean twoStepVerifi
   }
 
   /**
-   * @param isPartitionMovement True if partition movement per broker, false if the total leader movement.
+   * @param isInterBrokerPartitionMovement True if inter-broker partition movement per broker.
```

Comment:
1. Missing JavaDoc for `request`.
2. Can we also explain the meaning of parameter values to intention? -- e.g. 
```
<false,false> -> Concurrent leader movements
<true, false> -> XXX
<true, true> -> undefined
<false, true> -> XXX
```